Sources/TuistKit/Commands/InspectCommand.swift

```diff
@@ -4,7 +4,10 @@ struct InspectCommand: AsyncParsableCommand {
     static var configuration: CommandConfiguration {
         CommandConfiguration(
             commandName: "inspect",
```

Comment:
Let's add abstract to this command as well.

The entry is missing in `tuist --help`, which looks out of place:
![image](https://github.com/user-attachments/assets/8ced0495-15e8-4316-a529-d960ae8a3185)